doc/how-to/access_database_remotely.md

```diff
@@ -4,13 +4,39 @@
 done as a last resort. This should ideally only be done in pairs, and
 mutating any live data is STRONGLY discouraged.
 
+Both methods detailed below use [Jaqy](https://teradata.github.io/jaqy/), which
+is a Java-native universal database client. As well as carrying out simple 
+queries, you can also do exports of data (enabling, for example, one off reporting), 
+and other useful bits and pieces. For full details, check the [documentation](https://teradata.github.io/jaqy/).
+
+There are two ways to do this:
+
+## The easy way
+
+For most use cases (i.e. running simple queries etc), this will be the way
+you want to do it. The downside is you won't have access to things like
+data exports from the local shell.
+
 From your local machine run:
 
 ```bash
-script/remote_shell $ENVIRONMENT
+script/remote_db $ENVIRONMENT
 ```
 
-Where $ENVIRONMENT is the environment you want to connect to
+Where $ENVIRONMENT is the environment you want to connect to.
+
+When you're finished, just `ctrl+c` to end the session.
```

Comment:
Minor question/nitpick: what is `SIGINT` being sent to in this case? There's potentially processes/resources not being closed on the remote machine, so I'd say we should recommend using the `.quit` command to finish the session.

Reply:
Oh good call. I thought it wouldn't end the session for some reason, but I've just tried `.quit` and it works!